app/src/main/java/org/simple/clinic/home/patients/PatientsScreen.kt

```diff
@@ -102,6 +105,19 @@ open class PatientsScreen(context: Context, attrs: AttributeSet) : RelativeLayou
         .observeOn(mainThread())
         .takeUntil(screenDestroys)
         .subscribe { uiChange -> uiChange(this) }
+
+    illustrationImageView.setImageResource(illustrationResourceId())
+  }
+
+  private fun illustrationResourceId(): Int {
+    val today = LocalDate.now()
+    val may_11_2019 = LocalDate.of(2019, Month.MAY, 11)
```

Comment:
World hypertension day repeats on the same day every year, so we should not be checking for the year as well.

Otherwise, this piece of code will need to be continuously updated every year.

Reply:
done